### PR TITLE
Relase Candidate for v0.0.4

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Release that adds a method to obtain the IRTAM coefficients, improves the documentation, and fixes some bugs in the main library.

# Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality
      to not work as expected)
- This change requires a documentation update

# How Has This Been Tested?

Added unit tests and updated the documentation example.

**Test Configuration**:
* Operating system: OS X Big Sur
* Version number: Python 3.9
* Any details about your local setup that are relevant

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zenodo.json
- [x] doc tests pass locally
- [x] Upload the test distribution to the testPyPi server
- [x] Contacted developers and users about the future release
- [x] Performed integration tests with downstream packages (include list of these packages)
- [ ] Create a wiki page with the release text (put link here)
- [ ] After approval, confirm all changes from the RC pull are listed in the wiki release text.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

### CODE REVIEW

**Review by victoriyaforsythe — DISMISSED** [review]

File: PyIRTAM/coeff.py [hunk 1]
```diff
@@ -57,7 +59,8 @@ def download_irtam_coeffs(dtime, param, irtam_dir='', use_subdirs=True,
 
     # Determine the appropriate output filename
     param_file = get_irtam_param_filename(dtime, param, irtam_dir, use_subdirs)
-
+    print('Saved as: ', param_file)
```
Comment (aburrell) [inline PyIRTAM/coeff.py:62]: Print statements should be logging statements.

**Review by victoriyaforsythe — APPROVED** [review]

**Review by victoriyaforsythe — DISMISSED** [review]

File: PyIRTAM/lib.py [hunk 2]
```diff
@@ -1152,20 +1153,17 @@ def run_PyIRTAM(year, month, day, aUT, alon, alat, aalt, F107, irtam_dir):
         minute = int((aUT[it] - hour) * 60.)
         dtime = dt.datetime(year, month, day, hour, minute, 0)
 
+        # If the user passes '' for irtam_dir, the coefficients are
+        # downloaded form the UMass Lowell data base
+        if irtam_dir == '':
```
Comment (aburrell) [inline PyIRTAM/lib.py:1158]:
```suggestion
        # If the user specifies, the coefficients are
        # downloaded form the UMass Lowell data base
        if download:
```

File: PyIRTAM/lib.py [hunk 3]
```diff
@@ -1037,7 +1033,12 @@ def run_PyIRTAM(year, month, day, aUT, alon, alat, aalt, F107, irtam_dir):
     F107 : float
         User provided F10.7 solar flux index in SFU.
     irtam_dir : str
-        Place where IRTAM coefficients are on user's local computer.
+        Directory with IRTAM coefficients, or '' to use package directory.
+        (default='')
+    use_subdirs : bool
+        If True, adds YYYY/MMDD subdirectories to the filename path, if False
+        assumes that the entire path to the coefficient directory is provided
+        by `irtam_dir` (default=True)
```
Comment (aburrell) [inline PyIRTAM/lib.py:1041]:
```suggestion
        by `irtam_dir` (default=True)
    download : bool
        If True, downloads coefficient files (default=False)
```

File: PyIRTAM/lib.py [hunk 4]
```diff
@@ -1014,7 +1009,8 @@ def call_IRTAM_PyIRI(aUT, dtime, alon, alat, aalt, f2, f1, e_peak, es_peak,
     return F2_result, F1_result, E_result, Es_result, EDP_result
 
 
-def run_PyIRTAM(year, month, day, aUT, alon, alat, aalt, F107, irtam_dir):
+def run_PyIRTAM(year, month, day, aUT, alon, alat, aalt, F107, irtam_dir='',
+                use_subdirs=True):
```
Comment (aburrell) [inline PyIRTAM/lib.py:1013]:
```suggestion
                use_subdirs=True, download=False):
```

**Review by aburrell — COMMENTED** [review]
Made changes for downloads.

**Comment by aburrell** [comment]
Once we're done making changes I'll get the unit tests to finish clean.

**Review by victoriyaforsythe — APPROVED** [review]

**Pull request merged**